internal/cmd/cmd.go

```diff
@@ -124,24 +122,10 @@ func CreateControllerManager(config *controller.Config, logger logr.Logger) (
 	return csrController, mgr, 0
 }
 
-//
-func initLogger(config *controller.Config) logr.Logger {
```

Comment:
@hansedong this function was declared but not used, as you moved it later to `utils`. just removed it :)

Reply:
> @hansedong this function was declared but not used, as you moved it later to `utils`. just removed it :)

It has been resloved, thanks for pointing it out.